egui_web/src/backend.rs

```diff
@@ -223,8 +239,12 @@ impl AppRunner {
     }
 
     fn integration_info(&self) -> epi::IntegrationInfo {
+        #[cfg(not(feature = "use_glow_painter"))]
+        const NAME: &str = "egui_web";
+        #[cfg(feature = "use_glow_painter")]
+        const NAME: &str = "egui_web(painted by glow)";
         epi::IntegrationInfo {
-            name: "egui_web",
+            name: NAME,
```

Comment:
we could add a new field `painter: &str` to `IntegrationInfo` and set to either `webgl1`, `webgl2` or `glow` (`painter: self.painter.name()` if we add a `fn name(&self) -> &'static str` to `trait Painter`).

Reply:
…but perhaps in a separate PR